ide/coqide/wg_Find.ml

```diff
@@ -137,13 +139,16 @@ class finder name (view : GText.view) =
       end
 
     method private set_not_found () =
-      find_entry#misc#modify_bg [`NORMAL, `NAME "#F7E6E6"];
```

Comment:
Adding the icon and the tooltip are fine, but I would keep the reddish/pink background after an unsuccessful find because some users may prefer the current behavior.  GUI detail preferences are subjective.

Reply:
The part of it which is not subjective is bug #11024 which was caused by the background color. This is the reason to using icons instead.